cdk.json

```diff
@@ -31,13 +31,22 @@
       "aws-cn"
     ],
     "dev": {
-      "IMAGE_PATH_AND_TAG": "ghcr.io/sage-bionetworks/data_curator:0.2.12-beta",
+      "IMAGE_PATH_AND_TAG": "ghcr.io/sage-bionetworks/data_curator:0.2.13-beta",
       "AWS_DEFAULT_REGION": "us-east-1",
       "PORT": "3838",
       "COST_CENTER": "NO PROGRAM / 000000",
       "STACK_NAME_PREFIX": "dca",
       "ACM_CERT_ARN": "arn:aws:acm:us-east-1:631692904429:certificate/8644a975-2c1d-49ca-bc73-786745db0bc8",
       "VPC_CIDR": "10.255.70.0/24"
+    },
+    "prod": {
+      "IMAGE_PATH_AND_TAG": "ghcr.io/sage-bionetworks/data_curator:0.2.13-beta",
+      "AWS_DEFAULT_REGION": "us-east-1",
+      "PORT": "3838",
+      "COST_CENTER": "NO PROGRAM / 000000",
+      "STACK_NAME_PREFIX": "dca",
+      "ACM_CERT_ARN": "arn:aws:acm:us-east-1:878654265857:certificate/a3b7c804-c1fc-4e58-bbc6-541ef2d65816",
+      "VPC_CIDR": "10.255.70.0/24"
```

Comment:
should this be the same IP range as dev?

Reply:
@zaro0508 what do you think? We only mentioned the ARN when we met earlier.

Reply:
thanks, just updated that!